api/tests/test_views.py

```diff
@@ -760,3 +760,20 @@ def test_export_endpoint_validates_not_locked_shifts(
             prepared_ReportViewSet_view.check_for_not_locked_shifts(
                 second_months_report_locked_shifts
             )
+
+
+class TestDjoserCustomizing:
+    @pytest.mark.django_db
+    def test_delete_user_custom_serializer(self, user_object, user_object_jwt, client):
+        """
+        Test if the user-delete view works with the specified custom UserSerializer.
+        :param user_object:
+        :param user_object_jwt:
+        :param client:
+        :return:
+        """
+        client.credentials(HTTP_AUTHORIZATION="Bearer {}".format(user_object_jwt))
```

Comment:
Frage zum Verständnis: das loggt den Test ein und `user_object_jwt` wird als `fixture` in den Test gegeben?

Reply:
Correct.